examples/Custom_Hooks/probability_of_improvement.py

```diff
@@ -0,0 +1,149 @@
+## Plot the Probability of Improvement
+
+# This example demonstrates how to use the
+# {func}`register_hooks <baybe.utils.basic.register_hooks>` utility,
+# to calculate and analyse the probability of improvement:
```

Comment:
```suggestion
# to calculate and analyse the probability of improvement (PI) evolving during iterations:
```

once specified at the top its possible to only refer to `probability of improvement` as `PI`

Reply:
That is very convenient :)
Replaced it with `PI`, thanks!